netdeploy/remote/deployedNetwork.go

```diff
@@ -587,6 +587,12 @@ func accountsNeeded(appsCount uint64, assetCount uint64, params config.Consensus
 	var nAppAcct uint64
 
 	maxApps = uint64(params.MaxAppsCreated)
+	// TODO : given that we've added unlimited app support, we should revise this
+	// code so that we'll have control on how many app/account we want to create.
+	// for now, I'm going to keep the previous max values until we have refactored this code.
+	if maxApps == 0 {
+		maxApps = 10
+	}
 
 	if maxApps > 0 {
```

Comment:
This condition is alway true, since `maxApps` can no longer be 0.

Reply:
true; but I didn't want to change this logic at this time. I think that we will need more changes around this tool to align with the unlimited assets support, and I've just retained backward compatibility for now.